diffusers/training_example.ipynb

```diff
@@ -0,0 +1,3955 @@
+{
```

Comment:
<p>Is there any way to add some generated photos at the end of this notebook? Even if they're hard coded / loaded from elsewhere (for the same dataset/model).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Updated the code to do this...